.github/workflows/update-ami.yml

```diff
@@ -0,0 +1,51 @@
+name: Update AMI
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  create-pr:
+    name: Update AMI
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      id-token: write
+      
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Determine a release is marked as latest
+        id: get_release
+        run: |
+          LATEST_RELEASE_ID=$(gh release view --json id | jq '.id' | tr -d \")
+          CURRENT_RELEASE_ID=${{ github.event.release.node_id }}
+          
+          if [ $LATEST_RELEASE_ID == $CURRENT_RELEASE_ID ]; then
+            echo "release=latest" >> $GITHUB_OUTPUT
+          else
+            echo "release=other" >> $GITHUB_OUTPUT
+          fi
+        env:
+          GH_TOKEN: ${{ github.token }}          
+
+      - name: Assume AWS Promotion Role
+        if: steps.get_release.outputs.release == 'latest'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region:  us-west-2
+          # https://github.com/gravitational/teleport/issues/23066
+          role-to-assume: "arn:aws:iam::126027368216:role/tf-teleport-ami-gha-role"
+          role-session-name: "gha-docker-image-promotion-${{ github.run_number }}"
+          # Promotion should take roughly a minute, so 15 mins gives us plenty
+          # of leeway (and is the miminum value we can ask for).
+          role-duration-seconds: 900
+
+      - name: Create the PR
+        if: steps.get_release.outputs.release == 'latest'
+        run: |
+          TELEPORT_VERSION=$(make --no-print-directory print-version) make -C assets/aws create-update-pr
```

Comment:
What don't we take the version from the release event?

Reply:
Release tag, right?